rt-shell/objects/obj_shell/Draw_64.gml

```diff
@@ -204,6 +207,7 @@ if (isOpen) {
 		
 		draw_set_color(c_white);
 		draw_set_alpha(1);
+		draw_set_font(-1);
```

Comment:
what purpose does this serve?

Reply:
See the second part of #20 

Basically resets the font back to GM's default after we are done using it.

Reply:
Oh, I didn't even know GM had a default font! 😅